transactions/randomBeaconHistory/scripts/get_source_of_randomness.cdc

```diff
@@ -0,0 +1,15 @@
+import "RandomBeaconHistory"
+
+/// Retrieves the source of randomness for the requested block height from the RandomBeaconHistory Heartbeat
+///
+pub fun main(atBlockHeight: UInt64): [UInt8] {
+    // Get the account storing the heartbeat
+    let heartbeatAccount = getAccount(RandomBeaconHistory.getRandomBeaconHistoryAddress())
+    // Borrow the HeartbeatPublic Capability
+    let randomBeaconHistoryHeartbeat = heartbeatAccount.getCapability<&{RandomBeaconHistory.HeartbeatPublic}>(
+            RandomBeaconHistory.HeartbeatPublicPath
+        ).borrow()
+        ?? panic("Couldn't borrow RandomBeaconHistory.Heartbeat Resource")
+    // Return the source of randomness for the requested block height
+    return randomBeaconHistoryHeartbeat.sourceOfRandomness(atBlockHeight: atBlockHeight)
```

Comment:
This looks way to verbose! Can this be packaged into `RandomBeaconHistory.sourceOfRandomness(atBlockHeight: atBlockHeight)`.

Reply:
Sure, can do. I'll wire up the methods against the contract.